### PR TITLE
GDAP Documentation updates (beta APIs)

### PR DESCRIPTION
**Pull request by koravvams**

### CODE REVIEW

File: changelog/Microsoft.Partner.CustomerSvcAdmin.json [hunk 1]
```diff
@@ -185,6 +185,72 @@
...
+        {
+          "Id": "40f1802b-57ee-463b-98f0-3127c867310f",
+          "ApiChange": "EnumType",
+          "ChangedApiName": "longRunningOperationStatus",
+          "ChangeType": "Addition",
+          "Description": "Added the **longRunningOperationStatus** enumeration type.",
+          "Target": "longRunningOperationStatus"
+        },
+        {
+          "Id": "40f1802b-57ee-463b-98f0-3127c867310f",
```
Comment (Lauragra) [inline changelog/Microsoft.Partner.CustomerSvcAdmin.json:224]: For this and the following ChangeList record, I don't see the associated doc updates. Please remove, or add the updates to this PR.

**Comment by Lauragra** [comment]
@koravvams , please follow the Get started steps to set yourself up to be a contributor to the docs, so that you can follow our content workflow.

**Comment by koravvams** [comment]
#feedback-addressed

File: changelog/Microsoft.Partner.CustomerSvcAdmin.json [hunk 2]
```diff
@@ -185,6 +185,48 @@
...
+          "ChangeType": "Addition",
+          "Description": "Added the **succeeded** member to the **delegatedAdminRelationshipRequestStatus** enumeration.",
+          "Target": "delegatedAdminRelationshipRequestStatus"
+        },
+        {
+          "Id": "40f1802b-57ee-463b-98f0-3127c867310f",
+          "ApiChange": "Member",
+          "ChangedApiName": "complete",
+          "ChangeType": "Deletion",
+          "Description": "Removed the **complete** member from the **delegatedAdminRelationshipRequestStatus** enumeration.",
```
Comment (FaithOmbongi) [inline changelog/Microsoft.Partner.CustomerSvcAdmin.json:212]:
```suggestion
          "Description": "Removed the `complete` member from the **delegatedAdminRelationshipRequestStatus** enumeration.",
```

**Comment by koravvams** [comment]
> Thank you @koravvams for your work updating the docs to reflect the API changes. I've left suggestions for minor fixes and this PR will be good to go.

Thanks @FaithOmbongi for review and suggestions. I have addressed as per comments

**Comment by koravvams** [comment]
#feedback-addressed

**Comment by FaithOmbongi** [comment]
Removed the "changes requested" label. Bot might need a little bit of tweaking.

**Pull request merged**